builders/tools/precompiles/democracy.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Democracy 
+description: Moonbeam Democracy Solidity Precompile Interface Demo
+---
+
+# Interacting with the Democracy Precompile
+
+![Staking Moonbeam Banner](/images/builders/tools/precompiles/democracy/democracy-banner.png)
+
+## Introduction {: #introduction } 
+
+As a Polkadot parachain, Moonbeam features native on-chain governance thanks to the [substrate democracy pallet](https://marketplace-staging.substrate.dev/pallets/pallet-democracy){target=_blank}. The democracy precompile allows developers to access the governance functions of the substrate democracy pallet directly from a solidity interface. This enables a vastly improved end-user experience. For example, token-holders can vote on referenda directly from MetaMask, rather than importing an account in Polkadot.Js Apps and navigating a complex UI. 
+
+The democracy pallet is coded in Rust and it is part of a pallet that is normally not accessible from the Ethereum side of Moonbeam. However, the Democracy Precompile allows developers to access the native substrate democracy features using the Ethereum API in a precompiled contract located at address:
```

Comment:
I feel like this first sentence should be before the following sentence: "The democracy precompile allows developers to access the governance functions of the substrate democracy pallet directly from a solidity interface"

Not exactly sure but it seems like something needs to be rearranged so that you can kind of flow in the sense of like first list the problem with the way things currently are and then expand on how it solves the problem